source/manual/data-gov-uk-supporting-ckan.html.md

```diff
@@ -179,3 +179,97 @@ to purge the queues used in the various stages of harvesting
 paster --plugin=ckanext-harvest harvester purge_queues -c $CKAN_INI
 ```
 
+## Transferring data between CKAN instances
+
+In order to transfer data between CKAN instances, it is required to dump the content of the old instance into JSONL format, do a conversion to the new schema, then reload into a new instance.  This can be done using the CKAN API, by installing [ckanapi](https://github.com/ckan/ckanapi) on your local machine.
+
+For a local CKAN, you must append the path to the `ckan.ini` file:
+
+```
+ckanapi -c CKAN_INI ...
+```
+
+To administer a remote instance, you will require a sysadmin account, your username and an API key (obtained through the web interface).  Append the remote URL and API key to each ckanapi request, in place of the `-c` flag:
+
+```
+ckanapi -r URL -a API_KEY ...
+```
+
+#### Dumping data
+
+Active data (i.e. everything that is publicly available) can be retrieved through the API.  To dump data that is not available publicly, you must using an API key belonging to a sysadmin user.
+
+Dumping users:
+
+```
+ckanapi dump users --all -O USER_FILE.jsonl.gz -z -p 4
+```
+
+Dumping publishers:
+
+```
+ckanapi dump organizations --all -O ORGANIZATION_FILE.jsonl.gz -z -p 4
+```
+
+Dumping datasets:
+
+```
+ckanapi dump datasets --all -O DATASET_FILE.jsonl.gz -z -p 4
+```
+
+Retrieve the user data from Drupal (this is a nightly dumped file):
+
+```
+rsync -L --progress co@co-prod3.dh.bytemark.co.uk:/var/lib/ckan/ckan/dumps_with_private_data/drupal_users_table.csv.gz DRUPAL_USER_DUMP.csv.gz
+```
+
+#### Data schema migration
+
+The data schema for the two CKAN instances may not match.  Therefore a conversion is required to allow the dump to be imported to the new CKAN installation.  Scripts are available in `ckanext-datagovuk` to enable this.
```

Comment:
How about we link to https://github.com/alphagov/ckanext-datagovuk?

Reply:
Change made and rebased